experimenter/experimenter/experiments/tests/api/v5/test_serializers/mixins.py

```diff
@@ -0,0 +1,18 @@
+from unittest import mock
+
+
+class MockFmlErrorMixin:
+    def setUp(self):
+        super().setUp()
+
+        mock_fml_errors_patch = mock.patch(
+            "experimenter.features.manifests.nimbus_fml_loader.NimbusFmlLoader.get_fml_errors",
+        )
+        self.mock_fml_errors = mock_fml_errors_patch.start()
+        self.addCleanup(mock_fml_errors_patch.stop)
+
+    def setup_fml_no_errors(self):
+        self.mock_fml_errors.return_value = []
+
+    def setup_get_fml_errors(self, fml_errors):
+        self.mock_fml_errors.return_value = fml_errors
```

Comment:
I was following the precedent set by the other `mixins.py` (e.g. [`setup_kinto_pending_review`](https://github.com/mozilla/experimenter/blob/13fc2232f63c6b266ae7771c84a408ae4f6933b7/experimenter/experimenter/kinto/tests/mixins.py#L18) and [`setup_cached_sizing_data`](https://github.com/mozilla/experimenter/blob/13fc2232f63c6b266ae7771c84a408ae4f6933b7/experimenter/experimenter/jetstream/tests/mixins.py#L22C9-L22C33)). I don't think it's necessary to use `setup` instead of `set`, but it does make it internally consistent 😂 

I'd like to keep the `no_errors` separate, because we're using that in the `setUp` methods for `TestNimbusReviewSerializerSingleFeature` and `TestNimbusReviewSerializerMultiFeature` and I think it makes it a little more explicit to see `no_errors` vs `[]`. What do you think?

Reply:
I was unaware of the existing mixins, so feel free to ignore this feedback :)